Terminal.Gui/View/ViewLayout.cs

```diff
@@ -495,7 +495,7 @@ protected void ClearLayoutNeeded ()
 		public Point ScreenToView (int x, int y)
 		{
 			if (SuperView == null) {
-				return new Point (x - Frame.X, y - _frame.Y);
+				return new Point (x - _frame.X, y - _frame.Y);
```

Comment:
@tig I only was consistent here, because the old code was using `Frame.X` and `_frame.Y`. So I replaced the` Frame` to `_frame`. But I agree that is better to use the public accessor because it may have additional actions. I'll replace both with `Frame`.